pages/windows/cmd.md

```diff
@@ -3,34 +3,26 @@
 > The Windows command interpreter.
 > More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/cmd>.
 
-- Start a new instance of the command interpreter:
+- Start an interactive shell session:
 
 `cmd`
 
-- Run the specified command and then exit:
+- Execute a [c]ommand:
 
 `cmd /c "{{command}}"`
 
-- Run the specified command and then enter an interactive shell:
+- Execute a script:
+
+`cmd {{path/to/file.bat}}`
+
+- Execute a command and then enter an interactive shell:
 
 `cmd /k "{{command}}"`
 
-- Disable the usage of `echo` in command output:
+- Start an interactive shell session with disabled usage of `echo` in command output:
 
 `cmd /q`
 
-- Enable or disable command extensions:
+- Start an interactive shell session with enabled or disabled command [e]xtensions:
 
 `cmd /e:{{on|off}}`
-
-- Enable or disable file or directory autocompletion:
-
-`cmd /f:{{on|off}}`
-
-- Enable or disable environment variable expansion:
-
-`cmd /v:{{on|off}}`
-
-- Force output to use Unicode encoding:
-
-`cmd /u`
```

Comment:
Why to delete these examples, are they deprecated?

Reply:
I can return `cmd /u` example and `cmd /v:{{on|off}}`. But I don't know when disabling directory autocompletion can be useful. Also I have to note that phrase `Enable or disable environment variable expansion` is not almost correct: delayed expansion is toggled but not expansion at all. Sorry for deletion.

Reply:
Fine, then I think `cmd /f` could be deleted. And the "delay" term should be mentioned in the description of `cmd /v`.